app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -90,9 +91,20 @@ def refresh_targets_for_ems(ems, targets)
 
           Benchmark.realtime_block(:save_inventory) { save_inventory(ems, target, hashes) }
           _log.info "#{log_header} Refreshing target #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+
+          if hashes.kind_of?(Array)
+            _log.info "#{log_header} ManagerRefresh Post Processing #{target.class} [#{target.name}] id [#{target.id}]..."
+            # We have array of InventoryCollection, we want to use that data for post refresh
+            Benchmark.realtime_block(:manager_refresh_post_processing) { manager_refresh_post_processing(ems, target, hashes) }
+            _log.info "#{log_header} ManagerRefresh Post Processing #{target.class} [#{target.name}] id [#{target.id}]...Complete"
+          end
         end
       end
 
+      def manager_refresh_post_processing(_ems, _target, _inventory_collections)
```

Comment:
yeah, the old refresh is kind of disconnected from the processing, so only thing we can do (and are doing) are timestamp based queries to figure out what have changed